theforeman/foreman
# Fixes #37167 - Remove subscription entitlement report template

Drop subscription entitlement report template.

To test:
Run bundle exec rails db:migrate and make sure you don't see Subscription - Entitlement Report under Monitor > Report > Report Templates
